app/models/sample.rb

```diff
@@ -356,7 +356,7 @@ def kickoff_pipeline
     # The subsample field of "sample" is currently used as a simple flag (UI checkbox),
     # but was made an integer type in case we want to allow users to enter the desired number
     # of reads to susbample to in the future
-    pr.pipeline_branch = pipeline_branch.blank? ? "master" : pipeline_branch
+    pr.pipeline_branch = pipeline_branch.blank? ? "charles/hit-calling" : pipeline_branch
```

Comment:
Let's never change the default branch like this.   When a branch is ready to become default for an environment, it can be called release-{environment} (or until then, just master).  An admin can always special-request a branch.   We should avoid making versions of idseq-web and idseq-pipeline that interlock.

Reply:
It was meant as a temporary hack for letting Josh play around on alpha without needing to give him a long list of instructions. Never meant to be merged to master

Reply:
Thanks for turning it in something production-ready